pytest_timeout.py

```diff
@@ -66,29 +74,41 @@ def pytest_addoption(parser):
         choices=["signal", "thread"],
         help=METHOD_DESC,
     )
+    group.addoption(
+        "--timeout-signal-timer",
+        dest="timeout_signal_timer",
+        action="store",
+        choices=["ITIMER_REAL", "ITIMER_VIRTUAL", "ITIMER_PROF"],
+        help=SIGNAL_TIMER_DESC,
+    )
     parser.addini("timeout", TIMEOUT_DESC)
     parser.addini("timeout_method", METHOD_DESC)
     parser.addini("timeout_func_only", FUNC_ONLY_DESC, type="bool")
+    parser.addini("timeout_signal_timer", SIGNAL_TIMER_DESC)
 
 
 @pytest.hookimpl
 def pytest_configure(config):
     """Register the marker so it shows up in --markers output."""
     config.addinivalue_line(
         "markers",
-        "timeout(timeout, method=None, func_only=False): Set a timeout, timeout "
+        'timeout(timeout, method=None, func_only=False, signal_timer="ITIMER_REAL"): '
+        "Set a timeout, timeout "
         "method and func_only evaluation on just one test item.  The first "
         "argument, *timeout*, is the timeout in seconds while the keyword, "
         "*method*, takes the same values as the --timeout_method option. The "
         "*func_only* keyword, when set to True, defers the timeout evaluation "
         "to only the test function body, ignoring the time it takes when "
-        "evaluating any fixtures used in the test.",
+        "evaluating any fixtures used in the test."
```

Comment:
This is missing a trailing space, line concatenation will turn this into `test.The` currently afaik.

Reply:
Fixed